core/src/acc_text/nlg/gf/grammar.clj

```diff
@@ -6,18 +6,19 @@
             [clojure.string :as str]))
 
 (defn join-relation-ids [relations]
-  (->> relations
-       (map (comp #(str "x" %) name ::sg/to))
-       (str/join " ")))
+  (if (seq relations)
+    (->> relations
+         (map (comp #(str "x" %) name ::sg/to))
+         (str/join " "))
+    "\"\""))
```

Comment:
What is purpose of this empty quote?

Reply:
Sometimes, when concepts have no relations, similar grammar fragment will be produced: "Item. x01 ::= ;".
Instead, we want this line to look like this: "Item. x01 ::= \"\";" to be compiled successfully.